CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#4599](https://github.com/open-telemetry/opentelemetry-python/pull/4599))
 - Drop support for Python 3.8
   ([#4520](https://github.com/open-telemetry/opentelemetry-python/pull/4520))
+- Logging API accepts optional `context`; deprecates `trace_id`, `span_id`, `trace_flags`.
```

Comment:
Move to unreleased please

Reply:
Moved in [069c2f8](https://github.com/open-telemetry/opentelemetry-python/pull/4597/commits/069c2f864f063288647d998335585389ab680883)